### DISCUSSION TITLE
Export EXE failed

### DISCUSSION THREAD

**Discussion by aperfectline** [cables-gl/cables_docs#262] [Answers & Questions] [answered]
Hello I have tried to export an Exe, unfortunately it does not work.

I get the error message:

**build: All jobs have failed**

The repository on GitHub was created, but there is no Exe inside but, json. files and a patch folder

<img width="1181" alt="Screenshot 2022-06-03 11 10 39" src="https://user-images.githubusercontent.com/106805802/171826194-ef75ac5a-9484-4bf4-a854-e2636f785f4f.png">
<img width="1176" alt="Screenshot 2022-06-03 11 10 23" src="https://user-images.githubusercontent.com/106805802/171826203-ed2da815-fa74-4d8a-b58f-621638756b3f.png">
r

**Comment by steam0r** [comment]
would it be possible for you to invite me (steam0r) to that repo so i can have a look? it might be just github acting up and trying again might fix it, though

**Comment by aperfectline** [comment, marked as answer]
yes sure, i just added you

**Reply by steam0r** [reply]
github somehow failed to internally upload the successful build:

![Bildschirmfoto 2022-06-03 um 11 39 47](https://user-images.githubusercontent.com/151271/171834432-993e56d2-da34-4706-b56c-92f308e8f610.png)

rerunning the job manually (from the "Actions" tab), then ran and uploaded fine:

![Bildschirmfoto 2022-06-03 um 11 40 29](https://user-images.githubusercontent.com/151271/171834542-382221b8-1c2a-4b3c-871e-12571a95f44d.png)

you now have the executables on the release section of your repo:

![Bildschirmfoto 2022-06-03 um 12 09 36](https://user-images.githubusercontent.com/151271/171834720-cef2b434-0e36-4c6b-81c5-ce9ef52e465e.png)

**Reply by aperfectline** [reply]
thanks, i did a few more export runs, the behaviour is the same, the manual run works then.

do you have a guess where the problem lies with github?

**Reply by steam0r** [reply]
hm, will look into it with my account, weird...

**Reply by steam0r** [reply]
can you try this again **with a new repository**? i updated the way we upload these assets and had better results that way:



![Bildschirmfoto 2022-06-07 um 18 46 31](https://user-images.githubusercontent.com/151271/172437952-c5e954f3-d291-420e-9b37-a670b1994c03.png)

**Reply by aperfectline** [reply]
i have done 3 runs, all 3 have worked, thanks for the quick adjustment.
<img width="1236" alt="Screenshot 2022-06-07 21 14 52" src="https://user-images.githubusercontent.com/106805802/172464313-48c6e2b0-e828-4004-bccc-764d2838b124.png">

**Reply by steam0r** [reply]
cool, thanks for reporting and testing!
